### DISCUSSION TITLE
After logging in to the console, display 403(Access denied)

### DISCUSSION THREAD

**Discussion by zhouhao12** [minio/minio#18116] [Polls]
Sorry, my network does not allow me to ask questions in the community, so I can only ask questions here

After installing Minio using the Docker command, enter the console, http://XXX:20302/login，After successful login, the page becomes blank and F12 has no interface call

Found 403 Access denied only in the console and Docker has no logs

Can you help me take a look at this issue? Thank you.Please help me, thank you very much!

console log：
![image](https://github.com/minio/minio/assets/98568555/c2e0a6d9-bc00-4eb5-98c8-179b62d70edd)

The Docker command is as follows:
docker run 
-d 
-p 20302:9000 
-p 20303:9001 
--name=minio1 
--restart=always 
-e "MINIO_ROOT_USER=admin" 
-e "MINIO_ROOT_PASSWORD=admin123456" 
-v /home/smartpark/docker/minio1/data:/data 
-v home/smartpark/docker/minio1/config:/root/.minio  
minio/minio server /data --console-address ":9000" --address ":9001"

**Comment by ldc999** [comment]
I also encountered the same problem   
version: RELEASE.2023-11-11T08-14-41Z (go1.21.4 linux/amd64)

I am running the latest version of Minio in Docker, and initially logged into the console normally using HTTP.
Later on, I needed to log in to the console using the Minio TSL method ,and the Https method  ok, 
but I wanted to switch to HTTP，After removing the TSL related configuration, the HTTP login console page was blank and pressing F12 requested that the interface API/v1/session display Access Denied.

**Reply by ldc999** [reply]
![minio](https://github.com/minio/minio/assets/32814992/d6e815da-2f2a-45fc-aaff-c7f04b4dc334)
![1](https://github.com/minio/minio/assets/32814992/2a5f44b2-4b45-4286-b25d-485c9a352d89)

**Reply by gameplayerclash76-droid** [reply]
wowww

**Comment by changhungtao** [comment]
怎么解决的？我也遇到了同样的问题！

**Comment by john-morales** [comment]
I also ran into this issue, and found this poll to be a top google search result for the problem. I did find the solution, so sharing it here.

**TL;DR clear browser cookies**

To reiterate the steps the reproduce:

* minio installed with TLS configured
* successfully login to minio UI console
* shutdown minio, reconfigure minio without TLS, restart minio
* try and login to the console UI again
* observe the login attempt lands on a blank 403 page, with the `/api/v1/session` request returning a 403

Chrome Dev Tools for the `/api/v1/login` login request spells out the issue rather well when hovering over the warning icon for the `Set-Cookie` header in the response:

<img width="572" height="264" alt="image" src="https://github.com/user-attachments/assets/740193e5-3522-4b11-ad48-72d4b3254e70" />

This makes sense as the `/api/v1/session` request receives a 403 because it's not sending up the cookie value from `/api/v1/login`. Clear cookies for the page and try again and all is well. TIL about this Chrome behavior.
